build.gradle

```diff
@@ -28,5 +28,9 @@ task adrianExample (dependsOn: 'classes', type: JavaExec) {
 
 task runLangtonAntAdhemar (dependsOn: 'classes', type: JavaExec) {
     main = 'org.fundacionjala.at15.katas.langtonant.adhemar.Main'
-    classpath = sourceSets.main.runtimeClasspath
```

Comment:
Please also restore this line, it might break Adhemar's task

Reply:
Done!